README.md

```diff
@@ -0,0 +1,25 @@
+strong-pack
+===========
+
+Simple wrapper for `tar` for packaging a node application without ignoring
+anything except VCS files (`.git`, `.svn`, `.hg`, etc.).
```

Comment:
Ah, I think my original wording was a rephrase that referenced npm's .npmignore behaviour more explicitly. I'll reword.